ADRs/0001-adopt-multiaddr.md

```diff
@@ -0,0 +1,60 @@
+# ADR-0001: Adopt Multiaddr
+
+# Adopt Multiaddr for Addressing and Protocol Handling
+
+* Status: in review
+* Deciders: Pocket Network team
```

Comment:
Just for future convention, should this mention specifically the Protocol team?

Reply:
Pocket Network Protocol Team 

Will add it to the template